methods_and_flow_control/fizz_buzz.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+# # This program should output the numbers 1 through 100, however:
+# If evenly divisible by 3, print Fizz
+# If evenly divisible by 5, print Buzz
+# If evenly divisible by 3 and 5, print FizzBuzz
+# Otherwise, print out the number
+
+(1..100).each do |n|
+  r_3 = n % 3
+  output = 'Fizz' * (1 - (r_3 / (r_3 + 1.0))).floor
```

Comment:
You're making this much more complicated than necessary...  Perhaps an `if` condition would suffice?